test/integration/supported/client-pool-tests.js

```
@@ -3,6 +3,7 @@
 const { assert, expect } = require("chai");
 const dns = require("dns");
 const util = require("util");
+const { SocketAddress } = require("net");
```

ESLint

[error] 'SocketAddress' is assigned a value but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)